unidecode/__init__.pyi

```diff
@@ -0,0 +1,11 @@
+from typing import Any, Dict, Optional, Sequence
+
+Cache: Dict[int, Optional[Sequence[Optional[str]]]]
+
+class UnidecodeError(ValueError):
+    index: int = ...
```

Comment:
I think this should be `Optional[int]`, since constructor argument is also `Optional[int]`.

Reply:
Done. Good point.